artist/forms.py

```diff
@@ -12,8 +12,12 @@
 class ArtistApplyForm(forms.Form):
```

Comment:
Looks like the changes on this page should have been applied to https://github.com/RevolutionTech/perdiem-django/pull/363. Maybe you should apply the changes you've made here onto that PR and then we can revert this file.

Reply:
Made those updates so this should be good to be reverted.